plugins/node/instrumentation-mongoose/package.json

```diff
@@ -52,7 +52,7 @@
     "@types/node": "18.6.5",
     "expect": "29.2.0",
     "mocha": "7.2.0",
-    "mongoose": "6.11.5",
+    "mongoose": "^7.6.4",
```

Comment:
nit: The current latest is 7.6.6, so that could be bumped, but dependency automation will get it as well, so no worries.